tests/codemods/test_numpy_nan_equality.py

```diff
@@ -23,77 +23,114 @@ def test_simple(self, tmpdir):
         self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
         assert len(self.file_context.codemod_changes) == 1
 
-    def test_from_numpy(self, tmpdir):
+    def test_simple_inequality(self, tmpdir):
         input_code = """\
-        from numpy import nan
-        if a == nan:
+        import numpy
+        if a != numpy.nan:
             pass
         """
         expected = """\
         import numpy
+        if not numpy.isnan(a):
+            pass
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
 
-        if numpy.isnan(a):
+    def test_simple_inequality_2(self, tmpdir):
+        input_code = """\
+        import numpy
+        if not (a != numpy.nan):
+            pass
+        """
+        expected = """\
+        import numpy
+        if not (not numpy.isnan(a)):
```

Comment:
Simplifying boolean expressions feels like it might be a fun and useful codemod?

Reply:
Should make a fine codemod, specially as a transformation after codemods like this one.